docs/ensembl-help/using-ensembl/ensembl-apps/genome-browser/navigation.md

```diff
@@ -13,12 +13,32 @@ status: draft
 
 # Genome browser navigation
 
-You can navigate the Genome browser using mouse actions and the Navigation bar.
+You can navigate the Genome browser using a combination of the Navigation bar and mouse or track pad actions.
 
-To navigate with your mouse, move it into the browser image. You can move left or right by clicking and dragging the mouse. You can zoom in or out using the scroll wheel or two-finger scrolling on a trackpad – you will zoom towards the position of your cursor. The blue coordinates on the right of the Navigation bar indicate your current genomic position.
+## How to adjust position using the Navigation bar
 
-You can expand the Navigation bar by clicking on the blue coordinates. This will now display the chromosome with our region highlighted and some navigation icons. Zoom in and out using the plus and minus icons. Move left and right using the arrows. Click on change on the right to enter new genomic coordinates. Hide the navigation by clicking on the blue coordinates again.
+The blue coordinates on the right of the Navigation bar, next to the chromosome number, indicate your current genomic position in bp.
 
 ![The Navigation bar](navigation_bar.png)
 
-The coordinates of the Focus entity are shown on the left of the Navigation bar. As you move around using either the Navigation bar or the mouse actions, the Focus entity coordinates remain constant. To return to the region of your Focus entity, click on the target icon on the left, which will be blue when you are away from the Focus entity. When you are looking at the Focus entity, the target icon will be grey.
+You can expand the Navigation bar by selecting on the blue coordinates to reveal a set of navigation controls. 
```

Comment:
This is wrong, and the image is out of date. Suggest we remove this section until we have the new navigation panel ?